migration/data/course_tables.sql

```diff
@@ -498,6 +498,25 @@ CREATE TABLE regrade_discussion (
     deleted BOOLEAN DEFAULT FALSE NOT NULL
 );
 
+--
+-- Name: notifications_type_enum; Type: ENUM; Schema: public; Owner: -
+--
+CREATE TYPE notifications_type AS ENUM ('forum');
+
+--
+-- Name: notifications; Type: TABLE; Schema: public; Owner: -
+--
+CREATE TABLE notifications (
+    id serial NOT NULL PRIMARY KEY,
+    type notifications_type NOT NULL,
+    metadata TEXT NOT NULL,
+    content TEXT NOT NULL,
+    from_user_id VARCHAR(255) NOT NULL,
```

Comment:
we may want this to be null so that we can have notifications that don't originate from a user